### ISSUE TITLE
Make target `update_doc_table` has incorrect description

### ISSUE THREAD

**Issue by ckadner** [machine-learning-exchange/mlx#328]
The description of the Make target `update_doc_table: ## Check markdown files for invalid links` is incorrect (failed to update after copy/paste in PR #317)

https://github.com/machine-learning-exchange/mlx/blob/605fe331d863e62e770607130041d298268b4122/Makefile#L30-L31

It should read `## Regenerate the /docs/README.md file`

```Makefile
.PHONY: update_doc_table
update_doc_table: ## Regenerate the /docs/README.md file
```

The descriptions of the Make targets can be viewed when running `make help`  (or just `make` without specifying a target)

```Bash
$ make

help                      List the Make targets with description
check_npm_packages        Verify NPM packages
update_npm_packages       Update NPM packages
check_doc_links           Check markdown files for invalid links
update_doc_table          Regenerate the /docs/README.md file
check_license             Make sure source files have license header
```

**Comment by ezinneanne** [comment]
**Hello can I take this**

**Comment by ckadner** [comment]
Hi @ezinneanne -- yes, please go ahead and submit a PR for this. Thank you!

**Comment by ckadner** [comment]
Closed via #329
